server/routes/notifications.route.js

```diff
@@ -1,10 +1,11 @@
 import express from 'express';
 
-import notificationsCtrl from '../controllers/notifications.controller';
+import passErrors from '../helpers/passErrors';
+import { sendPushNotifications } from '../controllers/notifications.controller';
 
 const router = express.Router(); // eslint-disable-line new-cap
 
 router.route('/sendPushNotifications')
-    .post(notificationsCtrl.sendPushNotifications);
+    .post(passErrors(sendPushNotifications));
```

Comment:
Do we instead want to do something like this?

```js
// notifications.controller.js
import passErrors from '../helpers/passErrors';
const _sendPushNotifications = async (req, res) => {
    // do stuff
}
export const sendPushNotifications = passErrors(_sendPushNotifications);

// notifications.route.js
import { sendPushNotifications } from '../controllers/notifications.controller';
router.route('/sendPushNotifications').post(sendPushNotifications);
```

Reply:
Will do

Reply:
done!